<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupEditor</class>
 <widget class="QWidget" name="GroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
*
*         widget background (group_editor_wdg)
*
*/
#GroupEditor
{
	background-color: &quot;#292929&quot;;
	color: &quot;#DBDBDB&quot;;
}


/*
*
*          group_name_lbl
*
*/
#GroupEditor #group_name_lbl
{
	color: &quot;#EFEFEF&quot;;
}


/*
*
*          ctrl_btn (control buttons)
*
*/

/** normal  **/
#GroupEditor #ctrl_new_btn ,
#GroupEditor #ctrl_delete_btn ,
#GroupEditor #ctrl_validity_btn
{
	padding-top: 8;
	padding-bottom: 8;
	padding-left: 8;
	padding-right: 8;

	border-radius: 24;
	height: 32;

	background-color: &quot;#404040&quot;;
}

/** hover  **/
#GroupEditor #ctrl_new_btn:hover ,
#GroupEditor #ctrl_validity_btn:hover
{
	background-color: &quot;#707070&quot;;
}

#GroupEditor #ctrl_delete_btn:hover
{
	background-color: &quot;#794C4C&quot;;
}

/** pressed  **/
#GroupEditor #ctrl_new_btn::pressed ,
#GroupEditor #ctrl_delete_btn::pressed ,
#GroupEditor #ctrl_validity_btn::pressed
{
	background-color: &quot;#756588&quot;;
}

#GroupEditor #ctrl_delete_btn::pressed
{
	background-color: &quot;#902A2C&quot;;
}


/*
*
*          TableView
*
*/

/** table bg  **/
#GroupEditor #table_view QWidget
{
	border-radius: 8;
	background-color: &quot;#404040&quot;;					/* table bg*/
}


#GroupEditor #table_view
{

	border-left: none;
	border-right: none;

	gridline-color: transparent;
	spacing: 8;

	outline: none;
	background-color: &quot;#383838&quot;;
	border-radius: 8;
}



/** table items  **/
#GroupEditor #table_view::item
{
	padding: 4;

	background-color: transparent;
	border-bottom: 1px solid &quot;#707070&quot;;  /* Bottom border for row separation */
}

#GroupEditor #table_view::item:selected
{
	color: white;
	background-color: &quot;#756588&quot;;  /* Bottom border for row separation */

	border: none;
	outline: none;
}


QHeaderView::section 
{
    border-left: none;
    border-right: none;
}

/*
*
*          mapper widget
*
*/

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QLabel" name="group_name_lbl">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>32</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GroupName 1</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spc_top">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btn_control_hbox">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="ctrl_new_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>12</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string> Add New Path</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/add_icon.svg</normaloff>:/icons/add_icon.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ctrl_validity_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>12</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string> Check All Validity</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/refresh_icon.svg</normaloff>:/icons/refresh_icon.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ctrl_delete_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>12</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="text">
        <string> Delete Selection</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/delete_icon.svg</normaloff>:/icons/delete_icon.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ctrl_spc">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spc_mid">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableView" name="table_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="spc_btm">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="path_info_mapper" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
